splice/queries/reporting.py

```diff
@@ -0,0 +1,25 @@
+from splice.models import impression_stats_daily
+from splice.queries.tile import get_tiles_by_campaign
+from sqlalchemy.sql import func
+
+
+def get_stats_by_date(campaign_id):  # pragma: no cover
+
+    tile_ids = map((lambda tile: tile['id']), get_tiles_by_campaign(campaign_id))
+
+    from splice.environment import Environment
+
+    env = Environment.instance()
+    rows = (
+        env.db.session
+        .query(
+            impression_stats_daily.c.date,
+            func.sum(impression_stats_daily.c.impressions).label('impressions')
```

Comment:
If only fetching the `impressions` here, perhaps the better name would be `get_imps_by_date`. Ignore this one if more stats will be added in later on.